airflow/dags/ingestion/ask-astro-load.py

```diff
@@ -328,6 +329,20 @@ def extract_astro_blogs():
 
         return [df]
 
+    @task(trigger_rule="none_failed")
```

Comment:
WDYT about using the [`TriggerRule` enum ](https://github.com/apache/airflow/blob/ecb7e3a56e5f96bbbd3c6a7f97c40ba733bcc193/airflow/utils/trigger_rule.py#L23) instead?